tests/unit/test_build_helper.py

```diff
@@ -2728,6 +2743,21 @@ def test_copr_project_and_namespace(
             True,
             id="present-more-values",
         ),
+        pytest.param(
+            JobConfig(
+                type=JobType.copr_build,
+                trigger=JobConfigTriggerType.pull_request,
+                packages={
+                    "package": CommonPackageConfig(
+                        owner="the-owner",
+                        project="the-project",
+                    ),
+                },
+            ),
+            "something/different\ngit.instance.io/the/example/namespace/*",
```

Comment:
from a quick check I think we have here the test values incorrect as well, all these should be lists, should I have a look or could you cover it here?